paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/viewmodel/PaymentSheetViewModel.kt

```diff
@@ -27,7 +27,11 @@ internal class PaymentSheetViewModel(
 
             val checkoutResponse = runCatching {
                 repository.checkout(
-                    customer, Repository.CheckoutCurrency.USD, mode, false
+                    customer,
+                    Repository.CheckoutCurrency.USD,
+                    mode,
+                    setShippingAddress = false,
+                    setAutomaticPaymentMethods = false // need to verify this default
```

Comment:
@yuki-stripe Is this the same default you are using?

Reply:
Yes. The non-playground examples shouldn't use the playground backend.